CppImport/src/ClangHelpers.cpp

```diff
@@ -290,4 +293,57 @@ void ClangHelpers::attachDeclarationComments(clang::NamedDecl* namedDecl, Model:
 	}
 }
 
+void ClangHelpers::storeNodeFilename(Model::Node* node, clang::SourceLocation location)
```

Comment:
This name and the one after are not good. Store is a bad verb when all you do is insert something into a map. So definitely call these `insert ...`. Also I'm not sure having `filename` in the title makes sense. The client of this method doesn't seem to know anything about filenames.